### DISCUSSION TITLE
secondary structure file format

### DISCUSSION THREAD

**Discussion by John-zzh** [marrink-lab/vermouth-martinize#608] [Q&A]
Hi there, 
from the official tutorial I see an example ssd file is like
76
\~EEEEEETTS\~EEEEE\~\~TTSBHHHHHHHHHHHH\~\~\~GGGEEEEETTEE\~\~TTSBTGGGT\~\~TT\~EEEEEE\~\~S~\~
where can I find some resource introducing the meaning of these letters?

thx!

**Comment by csbrasnett** [comment]
I'm not sure what tutorial you're referring to, but the ssd files are generated by dssp during secondary structure assignment. For more details about dssp codes there's a page on their [site](https://pdb-redo.eu/dssp/about)

**Comment by John-zzh** [comment]
http://www.cgmartini.nl/index.php/tutorials-general-introduction-gmx5/proteins-gmx5

Thanks for response. This is the Martini tutorial I mentioned. And it has an example dssp file for 1UBQ.ssd,
http://www.[cgmartini.nl/images/stories/tutorial/2017/protein-tutorial-v2016.3/soluble-protein/ubiquitin/martini/1UBQ.ssd](http://www.cgmartini.nl/images/stories/tutorial/2017/protein-tutorial-v2016.3/soluble-protein/ubiquitin/martini/1UBQ.ssd)
